File: src/main/java/ru/stnk/RestTestAPI/dto/RolesDTO.java
package ru.stnk.RestTestAPI.dto;

// Класс для отправки на клиент роли (Data Transfer Object)
public class RolesDTO {

    private String nameRole;

    public RolesDTO() {
    }

    public RolesDTO(String nameRole) {
        this.nameRole = nameRole;
    }

    public String getNameRole() {
        return nameRole;
    }

    public void setNameRole(String nameRole) {
        this.nameRole = nameRole;
    }
}

File: src/main/kotlin/ru/stnk/resttestapi/dto/RolesDTO.kt
package ru.stnk.resttestapi.dto

// Класс для отправки на клиент роли (Data Transfer Object)
class RolesDTO {

    var nameRole: String? = null

    constructor() {}

    constructor(nameRole: String) {
        this.nameRole = nameRole
    }
}